### DISCUSSION TITLE
How to Integrate Consul with APISIX

### DISCUSSION THREAD

**Discussion by KalepuHarish** [apache/apisix#10957] [Q&A]
Hi am using Consul as service discovery and spring cloud gateway and it register as service in consul now i want migrate to apisix and i am using it apisix-docker 3.8 but i am unable to make connection between consul and apisix.

can someone please help and if any source or steps please help me

**Comment by kayx23** [comment]
Please see https://docs.api7.ai/apisix/how-to-guide/service-discovery/consul-integration
